data/interim/korea/KOR_health.csv

```diff
@@ -1,4 +1,4 @@
-adm1_name,date,adm0_name,cum_confirmed_cases,cum_deaths,cum_recoveries,active_cases
+adm1_name,date,adm0_name,cumulative_confirmed_cases,cumulative_deaths,cumulative_recoveries,cumulative_active_cases

```

Comment:
looks like we went back to "cumulative" from "cum". Can we use "cum_" and also change "cumulative_active_cases" to "active_cases"?